desc/compute/_stability.py

```diff
@@ -265,7 +265,10 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
     source_grid_requirement={"coordinates": "raz", "is_meshgrid": True},
     zeta0="array: points of vanishing integrated local shear to scan over. "
     "Default 15 points linearly spaced in [-π/2,π/2]",
+    # --no-verify Neigvals="int: Number of top eigenvalues to retain. "
+    # --no-verify "Default is 1",
 )
+# Neigvals above works for compute but not optimization
```

Comment:
what do you mean? if it does not work for some reason with optimization then just remove the kwargs etc entirely

Reply:
I get a "static attribute" error if I use constants["Neigvals"] instead of self._Neigvals.